src/assets/scripts/open-windows.bat

```diff
@@ -1,5 +1,5 @@
 @ECHO OFF
-      C:\Users\DCHAU\Documents\GitHub\workspace\dist\assets\scripts\nircmd.exe exec show "C:\Program Files\Mozilla Firefox\firefox.exe" /n
-    C:\Users\DCHAU\Documents\GitHub\workspace\dist\assets\scripts\nircmd.exe wait 500
-    C:\Users\DCHAU\Documents\GitHub\workspace\dist\assets\scripts\nircmd.exe win setsize foreground  480, 0, 1440, 270
+    start "" %1 /n
```

Comment:
Very nice job finding this command 👍! I thought we were screwed for a sec there b/c I could only open executables with the original method.